src/sentry/demo/apps.py

```diff
@@ -0,0 +1,10 @@
+from django.apps import AppConfig
+
+
+class Config(AppConfig):
+    name = "sentry.demo"
+
+    def ready(self):
+        from .tasks import build_up_org_buffer
+
+        build_up_org_buffer.apply_async()
```

Comment:
So just to make sure I understand this, you want this task to run every time the app is initialized, and then we'll rely on `assign_demo_org` to make sure we keep the buffer populated?

Reply:
@wedamija Correct